app/assets/stylesheets/custom.scss

```diff
@@ -441,9 +462,104 @@ ul.edit-listings  {
 //    =========================*/
 
 .listing-create-form {
-  background: #c5eaea;
+  background: rgba(131, 194, 177, 0.6);
 }
 
 .edit-new-listing {
   margin-top: 70px;
 }
+
+#nightly_rate {
```

Comment:
Avoid using id selectors<br>Selector `nightly_rate` should be written in lowercase with hyphens